tests/README.md

```diff
@@ -0,0 +1,47 @@
+# Integration Test for Image-rs
+
+This integration test has 2 main sub-type tests:
+* Image decryption using [ocicrypt-rs](https://github.com/confidential-containers/ocicrypt-rs)
+* Image signature verification.
+
+## Image Decryption
+
+Implemented in `decrypt_layer.rs`.
+
+Image decryption will involve the image `"docker.io/arronwang/busybox_kbs_encrypted"`,
+following the steps:
+
+* Pull manifefst of `"docker.io/arronwang/busybox_kbs_encrypted"` without verification of signature.
+* Pull layers of the mentioned image.
+* Ocicrypt-rs will ask the Attestation-Agent to decrypt the LEK, which is 
```

Comment:
KEK and Lek are not standard terms, so it is best to explain before using here.

Reply:
Sure!